zebra/rt_netlink.c

```diff
@@ -1240,6 +1252,44 @@ static bool _netlink_route_build_singlepath(const struct prefix *p,
 					      sizeof(label_buf)))
 		return false;
 
+	if (nexthop->nh_seg6local_ctx) {
```

Comment:
I see the code that uses srv6 data to _build_ nexthops, but no code that parses incoming notifications. what happens if an externally-configured nexthop is received - we won't process any associated srv6 info?

Reply:
Incoming notification is meaning "Non-FRR world like a iproute2" right?
In this implementation, we can't catch the external notification. :bow:

So

> what happens if an externally-configured nexthop is received - we won't process any associated srv6 info?

Yes, we will not configure for external noti.
I think, this is looked as future work...